docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -69,7 +69,7 @@ There are currently eight different Sproutlets:
 
 Each Sproutlet has three configuration options. The options have the same format for each Sproutlet, as listed here, with `<sproutlet>` replaced by the appropriate Sproutlet name:
 
-* `<sproutlet>` - The port that the Sproutlet listens on. The default value is 5054 for some Sproutlets (those enabled by default) and 0 for others (those disabled by default)
+* `<sproutlet>` - The port that the Sproutlet listens on. The default value depends on the Sproutlet. The I-CSCF sproutlet's default value is 5052, the BGCF sproutlet's default value is 5053, the S-CSCF sproutlet's default value is 5054, and the MMTel sproutlet's default value is 5055. All other Sproutlet's default to 0 (meaning that they are disabled by default)
```

Comment:
(Whether or not you agree with that, the last "Sproutlet's" should be "Sproutlets".)